python/mxnet/visualization.py

```diff
@@ -213,12 +213,12 @@ def plot_network(symbol, title="plot", save_format='pdf', shape=None, node_attrs
         input symbol names (str) to the corresponding tensor shape (tuple).
     node_attrs: dict, optional
         Specifies the attributes for nodes in the generated visualization. `node_attrs` is
-        a dictionary of Graphviz attribute names and values. For example,
+        a dictionary of Graphviz attribute names and values. For example::
```

Comment:
1.  Avoid inline literal quote in code block
2. Add leading and ending line break to code block:

        node_attrs={"shape":"oval","fixedsize":"false"}

Reply:
fixed